books/items.py

```diff
@@ -0,0 +1,12 @@
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class BooksItem(scrapy.Item):
+    # define the fields for your item here like:
```

Comment:
You should define the fields for your item here. For example, if you are scraping book data, you might want fields like `title`, `author`, `price`, etc. Each field should be defined using `scrapy.Field()`, similar to the commented example.

Reply:
this generated automaticly